### ISSUE TITLE
Can not revise same performance on DukeMTMC 

### ISSUE THREAD

**Issue by ZepengW** [haochenheheda/NFormer#9]

**Comment by ZepengW** [comment]
I use the config file same to Market1501 provided.
But I get 78.4(mAP) and 86.2 R-1.
In your paper, the mAP and R-1 with nformer is 83.5 and 89.4.
So is the config file for dukemtmc is different with market1051?

run command:
python3 tools/train.py --config_file='configs/softmax_triplet_with_center.yml' MODEL.DEVICE_ID "('0')" DATASETS.NAMES "('dukemtmc')" DATASETS.ROOT_DIR "('~/workplace/dataset')" OUTPUT_DIR "('./log/dukemtmc-nformer')"


log:
2022-10-21 23:22:36,387 reid_baseline.train INFO: Epoch[120] Iteration[20/224] Loss: 1.055, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:39,391 reid_baseline.train INFO: Epoch[120] Iteration[40/224] Loss: 1.054, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:42,403 reid_baseline.train INFO: Epoch[120] Iteration[60/224] Loss: 1.052, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:45,415 reid_baseline.train INFO: Epoch[120] Iteration[80/224] Loss: 1.051, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:48,429 reid_baseline.train INFO: Epoch[120] Iteration[100/224] Loss: 1.051, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:51,458 reid_baseline.train INFO: Epoch[120] Iteration[120/224] Loss: 1.051, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:54,501 reid_baseline.train INFO: Epoch[120] Iteration[140/224] Loss: 1.049, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:22:57,544 reid_baseline.train INFO: Epoch[120] Iteration[160/224] Loss: 1.048, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:23:00,590 reid_baseline.train INFO: Epoch[120] Iteration[180/224] Loss: 1.047, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:23:03,604 reid_baseline.train INFO: Epoch[120] Iteration[200/224] Loss: 1.046, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:23:06,572 reid_baseline.train INFO: Epoch[120] Iteration[220/224] Loss: 1.043, Acc: 1.000, Base Lr: 3.50e-06
2022-10-21 23:23:07,325 reid_baseline.train INFO: Epoch 120 done. Time per batch: 30.887[s] Speed: 464.2[samples/s]
2022-10-21 23:23:07,326 reid_baseline.train INFO: ----------
2022-10-21 23:24:19,715 reid_baseline.train INFO: Validation Results - Epoch: 120
2022-10-21 23:24:19,716 reid_baseline.train INFO: mAP: 69.0%
2022-10-21 23:24:19,716 reid_baseline.train INFO: CMC curve, Rank-1  :83.7%
2022-10-21 23:24:19,716 reid_baseline.train INFO: CMC curve, Rank-5  :92.3%
2022-10-21 23:24:19,716 reid_baseline.train INFO: CMC curve, Rank-10 :93.9%
2022-10-21 23:32:57,497 reid_baseline.train INFO: mAP: 77.1%
2022-10-21 23:32:57,497 reid_baseline.train INFO: CMC curve, Rank-1  :86.0%
2022-10-21 23:32:57,497 reid_baseline.train INFO: CMC curve, Rank-5  :92.2%
2022-10-21 23:32:57,497 reid_baseline.train INFO: CMC curve, Rank-10 :93.9%
2022-10-21 23:41:21,619 reid_baseline.train INFO: mAP: 78.3%
2022-10-21 23:41:21,620 reid_baseline.train INFO: CMC curve, Rank-1  :85.6%
2022-10-21 23:41:21,620 reid_baseline.train INFO: CMC curve, Rank-5  :92.5%
2022-10-21 23:41:21,620 reid_baseline.train INFO: CMC curve, Rank-10 :94.1%
2022-10-21 23:49:46,520 reid_baseline.train INFO: mAP: 78.4%
2022-10-21 23:49:46,520 reid_baseline.train INFO: CMC curve, Rank-1  :85.6%
2022-10-21 23:49:46,521 reid_baseline.train INFO: CMC curve, Rank-5  :92.1%
2022-10-21 23:49:46,521 reid_baseline.train INFO: CMC curve, Rank-10 :94.0%
2022-10-21 23:58:12,988 reid_baseline.train INFO: mAP: 78.4%
2022-10-21 23:58:12,989 reid_baseline.train INFO: CMC curve, Rank-1  :86.2%
2022-10-21 23:58:12,989 reid_baseline.train INFO: CMC curve, Rank-5  :92.3%
2022-10-21 23:58:12,989 reid_baseline.train INFO: CMC curve, Rank-10 :94.3%

**Comment by haochenheheda** [comment]
I think maybe the feature extractor is not well-trained. Please check the performance with only the feature extractor. The mAP should be about 76 without NFormer. 
The training of the feature extractor is copied from [this](https://github.com/michuanhaohao/reid-strong-baseline) repo.

**Comment by ZepengW** [comment]
Do you mean that I can use Res50 weights pretrained by https://github.com/michuanhaohao/reid-strong-baseline.
And could you provide your NFormer weights?

**Comment by giraffeooo** [comment]
Dear author, I have been studying related work recently. Could you please provide me with your training model weights? Thank you very much!
